src/waterfall/waterfall.h

```diff
@@ -160,7 +169,7 @@ class Waterfall : public QQuickPaintedItem
     void mouseColumnDepthChanged();
     void mouseMove();
```

Comment:
// TODO mouseMove and mousePosChanged seems redundant, clean up

Reply:
They are not the same thing.

Reply:
English indicates otherwise: 'move' is the same as 'delta position'
We need comments or better identifiers.

Reply:
Yes, mouseMove should be renamed to mouseDepth